pandas/core/arrays/datetimelike.py

```diff
@@ -1079,6 +1405,44 @@ def _evaluate_compare(self, other, op):
         result[mask] = filler
         return result
 
+    # --------------------------------------------------------------
+    # Reductions
+
+    def _reduce(self, name, skipna=True, **kwargs):
+        op = getattr(self, name, None)
+        if op:
+            return op(skipna=skipna)
+        else:
+            return super(DatetimeLikeArrayMixin, self)._reduce(
+                name, skipna, **kwargs
+            )
+
+    def min(self, skipna=True):
```

Comment:
do we need/want argmin/argmax?

Reply:
Perhaps as a followup? Actually we should get these for free from the base class. https://github.com/pandas-dev/pandas/issues/24382